XIPs/xip-55-passkey-identity.md

```diff
@@ -0,0 +1,166 @@
+
+1. Introduce `secp256r1` accounts and get to 100% adoption across the network
+2. Begin rolling out additional association types
+
+### App layer changes
+
+With this change, apps need to expect that an inbox may be associated with 0-N Ethereum addresses, and any number of additional association types. This is technically true today, since an inbox may change its recovery address and then revoke any associated accounts. However, it may be safe to assume that most developers aren't accounting for this edge case and assume there is at least 1 Ethereum account associated with an inbox.
+
+Associating a new account type would require a signature from an existing member of the inbox (either installation or account) and a signature from the new associated account.
+
+As additional account types are introduced to the protocol apps will want to expose those identities to the user. In some cases, they may want to resolve account addresses to names using naming protocols native to the chain (for example, looking up Solana accounts with SNS).
+
+Developers will need to choose which name to display to other users when multiple names are present. This is true today, as an Ethereum account can be associated with many ENS names, but this change may increase the urgency for protocol-level support for a user-specified name prioritization scheme.
+
+### Other considerations
+
+Here are some "gotchas" around XMTP's other signature types that need to be considered.
+
+#### Relying party identifiers
+
+Unlike XMTP's other signature types, the signatures of passkeys are bound to relying party identifiers to protect against phishing. The signing provider generally gets to dictate what this value is, so the signer will have to provide it along with the public key for XMTP to track.
+
+#### Installation IDs
+
+XMTP address verification in the inbox ID generation algorithm will need to be tweaked to account for the new p256 address type.
+
+## Rationale and alternatives
+
+### Why does XMTP need Passkeys?
+
+To authenticate messages and group membership, XMTP needs all clients to have a consistent way of validating that the installation that signed the message is a member of an inbox. This is how XMTP prevents impersonation on the network. If this were implemented without passkeys, and each new association was made equivalent to EVM addresses today, each additional account type would be a backward-incompatible change that would need to be coordinated across the network. Passkeys allow XMTP to only add one new top-level account used for authenticating messages.
+
+Passkeys were specifically introduced to allow for inboxes that don't have any Ethereum account associated. If the goal were to only allow for additional association types *on top* of a required Ethereum address, the "Additional association types" part of this proposal could be implemented without the "New account type" section.
+
+## Backward compatibility
+
```

Comment:
Another other backward compatibility info we want to provide -- beyond the LibXMTP compatibility info below?